<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSttExportWordReport</class>
 <widget class="QDialog" name="QSttExportWordReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_tree"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_grid">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_prev_next">
           <item>
            <widget class="QPushButton" name="m_btnPrev">
             <property name="text">
              <string>上一项</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_btnNext">
             <property name="text">
              <string>下一项</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_btn">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="label_FilePath">
         <property name="text">
          <string>导出路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_editFilePath">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnSelPath">
         <property name="text">
          <string>浏览</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnExport">
         <property name="text">
          <string>导出报告</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
